plugins/com.google.cloud.tools.eclipse.appengine.newproject/src/com/google/cloud/tools/eclipse/appengine/newproject/CreateAppEngineStandardWtpProject.java

```diff
@@ -50,8 +50,7 @@
   public void execute(IProgressMonitor monitor) throws InvocationTargetException, CoreException {
     SubMonitor progress = SubMonitor.convert(monitor, 100);
     
-    // todo just use getproject().getLocationUri()
-    URI location = config.getEclipseProjectLocationUri();
+    URI location = config.getProject().getRawLocationURI();
```

Comment:
Is there a reason you're using `getRawLocationURI()`? From the Javadocs: 

>  Since the returned location may contain unresolved variables, the resulting URI is typically only suitable for display. To access or manipulate the actual resource backing location, clients should obtain the resolved location using getLocationURI().

`getRawLocationURI()` doesn't do any interpolation of variables, etc. E.g., this could be `${workspace}/${work folder}/${blah}/${blah}/${blah}`.